apps/voting/app/src/hooks/useExtendedVoteData.js

```diff
@@ -15,31 +15,19 @@ export default function useExtendedVoteData(vote) {
   const tokenContract = useTokenContract()
 
   const canExecutePromise = useMemo(() => getCanExecute(vote, api), [api, vote])
-  const canExecute = usePromise(
-    canExecutePromise,
-    [vote && vote.voteId, api],
```

Comment:
@AquiGorka I'm 99% sure we don't need these extra dependencies in the `usePromise()`; if they change, the `useMemo()` should also refire and create a new promise (which will trigger the `usePromise()`).

Reply:
Yep, you are right, I did not update any of that code, just added a way to get a hold of the promise itself, this should be updated.

Reply:
Updated in 7842779 😄

Reply:
Although I know am curious if the `canExecutePromise` should be a dep here

Reply:
It's an implicit dependency in `usePromise()`